src/protocol/protocol.yml

```diff
@@ -684,6 +684,10 @@ BrowserContext:
       returns:
         artifact: Artifact
 
+    harExport:
```

Comment:
nit: this is ok, but there was nothing wrong with returning optional artifact from close.